src/preferences/dialog/dlgprefinterface.cpp

```diff
@@ -87,27 +88,19 @@ DlgPrefInterface::DlgPrefInterface(QWidget * parent, MixxxMainWindow * mixxx,
     }
 
     QString configuredSkinPath = m_pSkinLoader->getConfiguredSkinPath();
-    QIcon sizeWarningIcon(":/images/preferences/ic_preferences_warning.png");
     int index = 0;
     for (const QFileInfo& skinInfo : skins) {
-        bool size_ok = checkSkinResolution(skinInfo.absoluteFilePath());
-        if (size_ok) {
-            ComboBoxSkinconf->insertItem(index, skinInfo.fileName());
-        } else {
-            ComboBoxSkinconf->insertItem(index, sizeWarningIcon, skinInfo.fileName());
-        }
+        ComboBoxSkinconf->insertItem(index, skinInfo.fileName());
 
         if (skinInfo.absoluteFilePath() == configuredSkinPath) {
             m_skin = skinInfo.fileName();
             ComboBoxSkinconf->setCurrentIndex(index);
             // schemes must be updated here to populate the drop-down box and set m_colorScheme
             slotUpdateSchemes();
             skinPreviewLabel->setPixmap(m_pSkinLoader->getSkinPreview(m_skin, m_colorScheme));
-            if (size_ok) {
-                warningLabel->hide();
-            } else {
-                warningLabel->show();
-            }
+            checkSkinResolution(configuredSkinPath)
+                    ? warningLabel->hide()
+                    : warningLabel->show();
```

Comment:
TBH I find the ternary operator weird when used in conjunction with void expressions.
Why not this?

```suggestion
            if (checkSkinResolution(configuredSkinPath)) {
                warningLabel->hide();
            } else {
                warningLabel->show();
            }
```

Reply:
Both variants were in that file, I decided to keep the compact one.
Are there any drawbacks I'm not aware of?

Reply:
No, but its confusing IMHO, because usually you use this operator to assign one of two values:

```cpp
int i = someBool ? 1 : 2;
``

This is much more compact than the alternative and also avoid that a sloppy programmer can leave `i` uninitialized.

```cpp
int i;
if (someBool) {
    i = 1;
} else {
    i = 2;
}
```

Reply:
Good catch! This is indeed very uncommon usage, never seen it before and didn't notice it.

Reply:
alright I reverted that.

sorry for the double push -f.
just learned about git commit --fixup & rebase -i --autosquash ... and this time it worked. yeaij!

Reply:
> alright I reverted that.
> 
> sorry for the double push -f.
> just learned about git commit --fixup & rebase -i --autosquash ... and this time it worked. yeaij!

Cool. Personally, `git commit -m fixup`,`git rebase -i HEAD~5`, cut-and-paste the commit below the commit to fix and replace `pick` with `fixup` is faster for me (might just be muscle memory though)...